pom.xml

```diff
@@ -49,4 +53,59 @@
         </dependencies>
     </dependencyManagement>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>${maven.source.version}</version>
+                <executions>
+                    <execution>
+                        <id>attach-sources</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>jar-no-fork</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>${maven.javadoc.version}</version>
+                <executions>
+                    <execution>
+                        <id>attach-javadocs</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <quiet>true</quiet>
+                    <doclint>none</doclint>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>net.revelc.code.formatter</groupId>
+                <artifactId>formatter-maven-plugin</artifactId>
+                <version>${maven.formatter.version}</version>
+                <configuration>
+                    <configFile>formatter/VaadinJavaConventions.xml</configFile>
```

Comment:
flow-components and flow projects both link to the file hosted on GH, so I wonder if it's simpler to do the same here. ([example](https://github.com/vaadin/flow-components/blob/880a90e5da1250726cd1338ff5ffbbaac9d9940d/pom.xml#L479-L482))

Also, both define `<lineEnding>LF</lineEnding>` on their configuration.

Reply:
Even better then, so we don't need an additional module to store the file. Done!